wpilibc/athena/src/DriverStation.cpp

```diff
@@ -584,9 +582,13 @@ void DriverStation::Run() {
   while (m_isRunning) {
     HAL_WaitForDSData();
     GetData();
+    // notify IsNewControlData variables
+    m_newControlData = true;
```

Comment:
And like I said in my comment to #234, since one is a poll call and one is a blocking call, I don't see much of a use case for calling both from the same thread. We do guarantee that by the time either one is triggered, there will be new data to receive.